Training5/src/task1/src/components/Filter.js

```diff
@@ -37,32 +37,41 @@ const FilterItem = styled.p`
 
 export default function Filter() {
   const dispatch = useDispatch();
-  const filter = useSelector((state) => state.todo.filter);
-  console.log("filter", filter);
+  const filter = useSelector((state) => state.todo.isLoadCompleted);
 
-  function handleAllClick(){
-    const action = FilterAll();
+  function handleAllClick() {
+    const action = filterAll();
     dispatch(action);
   }
 
-  function handleCompletedClick(){
-    const action = FilterCompleted();
+  function handleCompletedClick() {
+    const action = filterCompleted();
     dispatch(action);
   }
 
-  function handleUncompletedClick(){
-    const action = FilterUnCompleted();
+  function handleUncompletedClick() {
```

Comment:
all functions like this should use useCallback to wrap it, I explain in the previous meeting